sre-recipes/sandboxctl

```diff
@@ -73,10 +78,29 @@ def get_recipes():
     recipes = get_valid_recipes(attributes)
     return recipes
 
+def restart_loadgen(pattern=LoadPattern.BASIC):
+    """
+    Restarts the load generator with the intent of deploying a
+    particular load pattern.
+    """
+    DEVNULL = open(devnull, 'w')
+    command = (
+        'sed -i -e'
+        ' "s/(basic|step):latest/step:latest/g"'
+        ' ../kubernetes-manifest/loadgenerator/locust-main-controller.yaml'
+        ' ../kubernetes-manifest/loadgenerator/locust-worker-controller.yaml'
+    )
+    subprocess.call(split(command), stdout=DEVNULL)
+    command = (
+        'kubectl apply'
+        ' -f ../kubernetes-manifest/loadgenerator/'
+    )
+    subprocess.call(split(command), stdout=DEVNULL)
+
 
 RECIPES = get_recipes()
 @click.command()
-@click.argument('action', type=click.Choice(['break', 'restore', 'verify']))
+@click.argument('action', type=click.Choice(['break', 'restore', 'verify', "loadgen-step"]))
```

Comment:
Does click support sub-commands? It might be better to make this `sandboxctl loadgen-pattern step` or something